packages/ui5-app/ui5-min.yaml

```diff
@@ -0,0 +1,56 @@
+specVersion: "2.0"
+metadata:
+  name: ui5-app
+type: application
+framework:
+  name: OpenUI5
+  version: "1.77.1"
+  libraries:
+    - name: sap.m
+    - name: sap.ui.core
+    - name: sap.ui.layout
+    - name: themelib_sap_fiori_3
+# https://sap.github.io/ui5-tooling/pages/extensibility/CustomServerMiddleware/
+server:
+  customMiddleware:
+  - name: ui5-middleware-servestatic
+    mountPath: /docs
+    afterMiddleware: compression
+    configuration:
+      debug: true
+      rootPath: ../../docs
+  - name: ui5-middleware-simpleproxy
+    mountPath: /proxy
+    afterMiddleware: compression
+    configuration:
+      debug: true
+      baseUri: "https://latest-openui5.rikosjett.com"
+      httpHeaders:
+        Any-Header: AnyHeader
+    # PoC: reuse the same middleware at a different "mountPath"
+  - name: ui5-middleware-simpleproxy
+    mountPath: /proxy2
+    afterMiddleware: compression
+    configuration:
+      baseUri: "https://latest-openui5.rikosjett.com"
+  - name: ui5-middleware-simpleproxy
+    mountPath: /ui5
+    afterMiddleware: compression
+    configuration:
+      baseUri: "https://sapui5.hana.ondemand.com/"
+      limit: "5mb"
+      removeETag: true
+  - name: ui5-middleware-cfdestination
+    afterMiddleware: compression
+    configuration:
+      debug: true
+      port: 1091
+      xsappJson: "xs-app.json"
+      destinations:
+      # check that the destination name (here: "backend") matches your router in xssppJson
+      - name: "backend"
```

Comment:
isn't it duplicate then? In Karma an here? But the servers are running on different ports, right?

Reply:
unfortunately karma(-ui5) doesn't play together with an external proxy running on a different port (yet), so need to duplicate the external proxy in karma conf itself.
see also: https://github.com/SAP/karma-ui5/issues/125

Reply:
OK, thanks